### PR TITLE
net-misc/freerdp: Require x11-libs/libXrandr if gstreamer and X Use flag are set 

### PR DESCRIPTION
**Pull request by fictitiousexistence**
Require x11-libs/libXrandr only if gstreamer and X flag are set since wayland systems don't use libXrandr.

### CODE REVIEW

**Comment by fictitiousexistence** [comment]
My apologies for opening / closing prs. Was having issues I didn't understand. Thanks.

File: net-misc/freerdp/freerdp-2.11.1.ebuild [hunk 1]
```diff
@@ -52,7 +52,9 @@ RDEPEND="
 	gstreamer? (
 		media-libs/gstreamer:1.0
 		media-libs/gst-plugins-base:1.0
-		x11-libs/libXrandr
+		X? (
+			x11-libs/libXrandr
+		)
```
Comment (floppym) [inline net-misc/freerdp/freerdp-2.11.1.ebuild:57]: Please collapse this to a single line.
```suggestion
		X? ( x11-libs/libXrandr )
```

**Review by floppym — CHANGES_REQUESTED** [review]
Changing RDEPEND will require revbumping each ebuild. Please do that via renames.

Please rebase on current master and include the newly added 3.4.0 ebuild.

**Comment by fictitiousexistence** [comment]
@floppym  Is this correct? I synced again since you pushed #35659 and renamed the r1's to r2. And made the single line change requested. Thanks.

**Pull request closed without merging**